robottelo/cli/base.py

```diff
@@ -443,7 +440,7 @@ def _construct_command(cls, options=None):
             elif val is not False:
                 if isinstance(val, list):
                     val = ','.join(str(el) for el in val)
-                tail += u' --{0}="{1}"'.format(key, val)
+                tail += u" --{0}='{1}'".format(key, val)
```

Comment:
It doesn't seem as impacting any tests, but failing for current suite, so after some re-tests decided to use that idea

hammer -v -u admin -p changeme --output=csv sc-param update --override="1" --id="3" --default-value="${$ntp::params::config_test}"
-bash: ${$ntp::params::config_test}: bad substitution

hammer -v -u admin -p changeme --output=csv sc-param update --override="1" --id="3" --default-value='${$ntp::params::config_test}'
Message,Id
Parameter updated,3